OpenQA/Isotovideo/Backend.pm

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018-2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::Isotovideo::Backend;
+use Mojo::Base -base, -signatures;
+
+use Mojo::File qw(path);
+use backend::driver;
+use bmwqemu;
+
+sub new ($class) {
+    $bmwqemu::vars{BACKEND} ||= "qemu";
+    $bmwqemu::backend = backend::driver->new($bmwqemu::vars{BACKEND});
+
+    path('os-autoinst.pid')->spurt("$$");
+    # might throw an exception
+    $bmwqemu::backend->start_vm;
+}
+
+has process => sub {
+    return $bmwqemu::backend->{backend_process};
+};
```

Comment:
Should this really be cached in `$self->{process}`? Seems like it could leak. A simple getter method might be more appropriate.

Reply:
I don't know what you mean by leak. It was previously stored in a variable. `stop` needs to do the cleanup like before.

Reply:
You're creating a new reference to the value behind `$bmwqemu::backend->{backend_process}` here. If that value ever changes and would have been garbage collected, the new reference would still exist and prevent garbage collection. That's why you generally only want to create these extra references via attribute accessors when you really need them. Which i don't believe to be the case here. Instead you could just do `sub process { $bmwqemu::backend->{backend_process} }`.